### ISSUE TITLE
Props being passed to input produces react warnings

### ISSUE THREAD

**Issue by rfviolato** [leonardowf/react-simple-currency#8]
Hello! I've been using this lib and I stumbled upon a problem, it seems that it's passing some props to the input tag so react produces warnings about it because they are not the html node's known props.

```
Warning: Unknown props `separator`, `delimiter`, `precision`, `unit`, `onInputChange`
on <input> tag. Remove these props from the element. For details, see
https://fb.me/react-unknown-prop
```

I think that [this line](https://github.com/leonardowf/react-simple-currency/blob/master/src/SimpleCurrencyInput.js#L98) is causing the problem.

Nice lib! :D

**Comment by gss-patricia** [comment]
The same issue for me

**Comment by leonardowf** [comment]
Problem solved on https://github.com/leonardowf/react-simple-currency/pull/9
Thanks @wfuertes
Released under v1.0.12
